sky/server/requests/serializers/encoders.py

```diff
@@ -121,7 +121,15 @@ def encode_status_kubernetes(
 
 
 @register_encoder('jobs.queue')
-def encode_jobs_queue(jobs_or_tuple):
+def encode_jobs_queue(jobs: List[dict],) -> List[Dict[str, Any]]:
```

Comment:
For example, we still accept `jobs_or_tuple` here, but transform it to `List[Dict[str, Any]]`

Reply:
Refer to comment https://github.com/skypilot-org/skypilot/pull/7058#discussion_r2329303973